docs/source/api/calls.rst

```diff
@@ -42,14 +42,15 @@ getNextPage(nextPageUri)
     :returns: {Promise<oobject>} Returns a promise that resolves to the next page of calls
     :throws: Will throw an error on a filed response.
 
-create(to, from, options)
+create(to, from, applicationId, options)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
     Create a new call through the Persephony API.
 
     :to: {string} The number to call out to (DNIS). This can be any valid phone number formatted in E.164 format in Persephony's service area.
     :from: {string} The number to call from (ANI). This must be a number purchased from Persephony or a verified phone number owned by the user.
-    :options: {object} Additional properties to set the behavior of the call to be placed. Must include either :code:`callConnectUrl` or :code:`applicationId`.
+    :applicationId: {string} The id of the application Persephony should use to handle the phone call.
+    :[options]: {object} Additional properties to set the behavior of the call to be placed. Must include either :code:`callConnectUrl` or :code:`applicationId`.
```

Comment:
the note about the callConnectUrl and applicationId should be removed from the options

Reply:
Excellent catch! I love code review.